src/ut_main.c

```diff
@@ -44,14 +44,15 @@ static optionFlags_t gOptions;  /*!< Control flags, should not be exposed outsid
 
 #define UT_LOG_DEFAULT_PATH "/tmp/"
 
-
 /* Function prototypes */
 
 static void usage( void )
 {
     TEST_INFO(( "-c - Console Mode (Default)\n" ));
     TEST_INFO(( "-a - Automated Mode\n" ));
     TEST_INFO(( "-b - Basic Mode\n" ));
+    TEST_INFO(( "-d - Disable Group \n" ));
+    TEST_INFO(( "-e - Enable Group \n" ));
```

Comment:
you need too print the groups in the help, so you know what to enable or disable.